unfoldingWord/translationCore
# init git repo, little cleanup for git module

#### This pull request addresses:

Uploading to D43 should now work.
Git Init is run before each save happens, which after research is harmless.
Addresses QA Fail for #974 

#### How to test this pull request:

Upload projects that do not have a .git folder in them, which should be most after our refactor.
To be sure, try on any USFM imported project.

Check QA issues #974 
Hand for the link, space between text and link.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2184)
